v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/dialectadapter/mysql/MysqlDialectAdapter.java

```diff
@@ -406,6 +406,79 @@ private String normalizeColumnType(String columnType) {
     return mySQlTypeAliases.getOrDefault(normalizedType, normalizedType);
   }
 
+  private String addWhereClause(String query, ImmutableList<String> partitionColumns) {
+
+    // Implementation detail, using StringBuilder since we are generating the query in a loop.
+    StringBuilder queryBuilder = new StringBuilder(query);
+
+    boolean firstDone = false;
+    for (String partitionColumn : partitionColumns) {
+
+      if (firstDone) {
+        // Add AND for iteration after first.
+        queryBuilder.append(" AND ");
+      } else {
+        // add `where` only for first iteration.
+        queryBuilder.append(" where ");
```

Comment:
nit - use same case for all clauses for uniformity.

Reply:
Fixed. Thanks.